modules/logging-loki-retention.adoc

```diff
@@ -26,7 +30,7 @@ metadata:
   name: logging-loki
   namespace: openshift-logging
 spec:
-  limits: 
+  limits:
    global: <1>
       retention: <2>
         days: 20
```

Comment:
BTW, if we want to target this content on for Logging 6.0, it's better to use v13 schema on the lokistack instead of v11.